### PR TITLE
Code cleaning of apps package

### PR DESCRIPTION
**Pull request by asanin-epfl**
standalone `apps` package have been moved under `neurom` package, refactored and covered with tests.

### CODE REVIEW

File: neurom/apps/cli.py [hunk 1]
```diff
@@ -4,18 +4,20 @@
...
-logger.setLevel(logging.INFO)
-
 
 @click.group()
-def cli():
+@click.option('-v', '--verbose', count=True, default=0,
+              help='-v for WARNING, -vv for INFO, -vvv for DEBUG')
+def cli(verbose):
     """The CLI entry point."""
+    level = (logging.WARNING, logging.INFO, logging.DEBUG, logging.DEBUG)[min(verbose, 2)]
```
Comment (tomdele) [inline neurom/apps/cli.py:19]: one extra debug

File: neurom/apps/cli.py [hunk 2]
```diff
@@ -42,3 +44,46 @@ def view(input_file, plane, backend, realistic_diameters):
 
     if backend == 'matplotlib':
         plt.show()
+
+
+@cli.command(short_help='Morphology statistics extractor, more details at'
+                        'https://neurom.readthedocs.io/en/latest/morph_stats.html')
+@click.argument('datapath', required=False)
+@click.option('-C', '--config', type=click.Path(exists=True, dir_okay=False),
```
Comment (tomdele) [inline neurom/apps/cli.py:52]: Is there a reason why you are using a capital letter for the `-C` ?
I see the `-I` also (capital i) for the ignored-exceptions.

File: neurom/apps/morph_check.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2015, Ecole Polytechnique Federale de Lausanne, Blue Brain Project
```
Comment (tomdele) [inline neurom/apps/morph_check.py:1]: I think this should be 
`2015-2021`

File: neurom/apps/morph_stats.py [hunk 4]
```diff
@@ -256,5 +268,48 @@ def sanitize_config(config):
...
+    """
+    if is_full_config:
+        config = full_config()
+    else:
+        try:
+            config = get_config(config, EXAMPLE_CONFIG)
+            config = sanitize_config(config)
+        except ConfigError as e:
+            L.error(e)
+            raise
```
Comment (tomdele) [inline neurom/apps/morph_stats.py:290]: I am still puzzled by the bare raise code wise ... 
But this is perfectly ok.

File: neurom/apps/morph_stats.py [hunk 5]
```diff
@@ -256,5 +268,48 @@ def sanitize_config(config):
...
+
+    results = {}
+    if as_population:
+        results[datapath] = extract_stats(neurons, config)
+    else:
+        for neuron in tqdm(neurons):
+            results[neuron.name] = extract_stats(neuron, config)
+
+    if not output_file:
+        print(json.dumps(results, indent=2, separators=(',', ':'), cls=NeuromJSON))
```
Comment (tomdele) [inline neurom/apps/morph_stats.py:305]: is the print intentional ?

File: neurom/apps/cli.py (hunk 1 above)
Comment (asanin-epfl) [reply neurom/apps/cli.py:19]: one scotch, one bourbon, one debug!

Comment (tomdele) [reply neurom/apps/cli.py:19]: one debug, 2 bourbons, one whiskey, one ... , .... zzzzzzz

Comment (arnaudon) [reply neurom/apps/cli.py:19]: guys, I can't stay focused with all this noise!

File: neurom/apps/cli.py (hunk 2 above)
Comment (asanin-epfl) [reply neurom/apps/cli.py:52]: backward compatibility only

File: doc/source/migration_v2.rst [hunk 6]
```diff
@@ -19,3 +19,5 @@ Migration to v2 version
     The following is not an invalid morphology anymore:
     - 2 point soma
     - non-sequential ids
+- script ``morph_check`` and ``morph_stats`` changed to ``neurom check`` and ``neurom stats``
```
Comment (arnaudon) [inline doc/source/migration_v2.rst:22]: this is used in morph_service i think, be careful here (@jdcourcol knows more on that)

File: neurom/apps/cli.py (hunk 1 above)
Comment (tomdele) [reply neurom/apps/cli.py:19]: have a drink then ...

File: neurom/apps/morph_check.py (hunk 3 above)
Comment (asanin-epfl) [reply neurom/apps/morph_check.py:1]: In all source files this line used. Should we update all other files as well?

**Review by arnaudon — APPROVED** [review]

File: neurom/apps/morph_stats.py (hunk 5 above)
Comment (asanin-epfl) [reply neurom/apps/morph_stats.py:305]: Yes, it is the intended behavior

File: neurom/apps/morph_check.py (hunk 3 above)
Comment (tomdele) [reply neurom/apps/morph_check.py:1]: If I recall correctly (from Andrew like a year ago), I ~~think we should update if the file changed during the year.
For not updated files I think this is fine.

Comment (asanin-epfl) [reply neurom/apps/morph_check.py]: @ASVG please clarify on copyright notices

**Review by tomdele — APPROVED** [review]
Looks good

**Comment by asanin-epfl** [comment]
Thank you!

**Pull request merged**